src/__tests__/directLine.test.ts

```diff
@@ -0,0 +1,19 @@
+import * as DirectLineExport from '../directLine';
+
+test('#setConnectionStatusFallback', () => {
+    const { DirectLine } = DirectLineExport;
+    expect(typeof DirectLine.prototype.setConnectionStatusFallback).toBe('function')
+    const { setConnectionStatusFallback } = DirectLine.prototype
```

Comment:
Semi-colon (and below).
```suggestion
    const { setConnectionStatusFallback } = DirectLine.prototype;
```